knowledge-base/grid-csv-export-change-field-delimiter.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Grid export to CSV - change the default field delimiter (comma)
+description: How to export the Grid to CSV with a different/custom field delimiter?
+type: how-to
+page_title: Grid export to CSV - change the default field delimiter (comma)
+slug: grid-kb-csv-export-change-field-delimiter
+position: 
+tags: grid, export, csv, field, delimiter, change, semicolon, comma, custom
+ticketid: 1576816
+res_type: kb
+---
+
+## Environment
+<table>
```

Comment:
Use spaces instead of tabs.

Reply:
Are you referring to the table? This is automatically created section from the admin.